src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2700,19 +2700,19 @@
     <value>Toggle group sort direction</value>
   </data>
   <data name="NewTabDescription" xml:space="preserve">
-    <value>Open a new tab</value>
+    <value>Open a new tab in the same window</value>
   </data>
   <data name="NavigateBackDescription" xml:space="preserve">
-    <value>Navigate backward</value>
+    <value>Navigate to the previous location</value>
   </data>
   <data name="NavigateForwardDescription" xml:space="preserve">
-    <value>Navigate forward</value>
+    <value>Navigate to the next location</value>
   </data>
   <data name="NavigateUpDescription" xml:space="preserve">
-    <value>Navigate up one directory</value>
+    <value>Navigate one level up in the folder tree</value>
```

Comment:
@Jay-o-Way `Navigate one level up in the folder tree` vs `Navigate one level up the folder tree`?

Reply:
Hm, tricky one, mostly looking at "navigate". That one is often used for websites and such. Simply saying "move one level up" sounds smoother. As for "in" or not, i would personally leave it out. Thinking of phrases like "move up a ladder".

In case of doubt, search MS Learn.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/folder-folder-icon
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui